include/assimp/SceneCombiner.h

```diff
@@ -223,7 +223,7 @@ class ASSIMP_API SceneCombiner {
      *    deletes the input scenes afterwards. There may be duplicate scenes.
      *  @param flags Combination of the AI_INT_MERGE_SCENE flags defined above
      */
-    static void MergeScenes(aiScene **dest, aiScene *master,
+    static void MergeScenes(aiScene **dest, std::unique_ptr<aiScene>& master,
```

Comment:
This is part of the public API, you can't change it (without major version bump). It's already documented as taking ownership of the pointer so just use `.release()` at the call site.

Reply:
Store it in a `unique_ptr` first thing inside `MergeScenes`?